client/src/pages/create-user/index.jsx

```diff
@@ -0,0 +1,28 @@
+import PropTypes from "prop-types";
+import CreateComponent from "../../components/create-user-component";
+
+function Create({ setManagedUsers }) {
+  const getNextAvailableUserId = (managedUsers) => {
+    const highestId = managedUsers.reduce((maxId, user) => {
+      return user.userID > maxId ? user.userID : maxId;
+    }, 0);
+
+    return highestId + 1;
+  };
+
+  const addUser = (newUser) => {
+    setManagedUsers((prevUsers) => {
+      const newUserID = getNextAvailableUserId(prevUsers);
+      const newUserWithID = { ...newUser, userID: newUserID };
+      return [...prevUsers, newUserWithID];
+    });
+  };
+  return <CreateComponent onAddUser={addUser} />;
+}
```

Comment:
I think you can remove these methods as well as the setManagedUsers state given navigating to the admin dashboard after creating a user will trigger a reload of managedUsers anyways